slack/ecs-deployment-failure/lambdas/main.rb

```diff
@@ -0,0 +1,98 @@
+require 'json'
+require 'net/http'
+
+FAILURE_EVENT_NAME = 'SERVICE_DEPLOYMENT_FAILED'.freeze
+
+def handler(event:, context:)
+  puts "[INFO] #{event}"
+  puts "[INFO] Context: #{context}"
+  event_name = event.fetch('detail').fetch('eventName')
+
+  return unless deployment_failure?(event_name)
+
+  resource_name = event.fetch('resources').join(',')
+  region = event.fetch('region')
+  reason = event.fetch('detail').fetch('reason')
+  _, cluster_name, service_name = resource_name.split('/')
+
+  response = post_to_slack(cluster_name, service_name, region, reason)
+  puts "[INFO] Response: #{response.body}"
+end
+
+def deployment_failure?(event_name)
+  event_name == FAILURE_EVENT_NAME
+end
+
+def post_to_slack(cluster_name, service_name, region, reason)
+  webhook_url = ENV['SLACK_WEBHOOK_URL']
+  uri = URI(webhook_url)
```

Comment:
```suggestion
  uri = URI(ENV.fetch('SLACK_WEBHOOK_URL'))
```

nit: why do we need to dereference the ENV var when we use it only once in the next line?